src/include/compat/pg_compat.h

```diff
@@ -360,14 +370,16 @@ extern void create_plain_partial_paths(PlannerInfo *root,
 #define ExecEvalExprCompat(expr, econtext, isNull, errHandler) \
 	ExecEvalExpr((expr), (econtext), (isNull))
 #elif PG_VERSION_NUM >= 90500
-#include "partition_filter.h"
 
 /* Variables for ExecEvalExprCompat() */
 extern Datum			exprResult;
 extern ExprDoneCond		isDone;
 
 /* Error handlers */
-static inline void mult_result_handler() { elog(ERROR, ERR_PART_ATTR_MULTIPLE_RESULTS); }
+static inline void mult_result_handler()
+{
+	elog(ERROR, "partitioning expression should return single value");
```

Comment:
I guess `ERR_PART_ATTR_MULTIPLE_RESULTS` is useless now, right?

Reply:
Yes. As commit message says, inclusion of partition_filter.h created dependency hell here.